modin/numpy/test/test_array.py

```diff
@@ -0,0 +1,778 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
```

Comment:
Should we split this file into multiple depending on the type operation? This file is really huge even for now.

Reply:
Makes sense to me!